docs/configuration.asciidoc

```diff
@@ -134,6 +134,11 @@ Note however that this can have adverse effects on performance.
 The name of the environment this service is deployed in,
 e.g. "production" or "staging".
 
+NOTE: `environment` is currently not supported in the Kibana APM UI.
+This means that data from different environments that share the same
+`<<config-service-name, service name>>` will be shown together in the APM UI.
```

Comment:
```suggestion
`<<config-service-name,service name>>` will be shown together in the APM UI.
```

Reply:
Not sure if this actually makes a difference in the rendered HTML, but I think it's a better syle.